articles/openshift/howto-using-azure-redhat-openshift.md

```diff
@@ -132,9 +97,14 @@ Follow these steps to create a virtual network containing two empty subnets.
 
 ## Create a cluster
 
-Run the following command to create a cluster.
+Run the following command to create a cluster. 
+
+* The '-g $RESOURCEGROUP' parameter refers to the OpenShift resource and can be used by multiple clusters.
+* The '--cluster-resource-group "aro-$CLUSTER' parameter referes to a new resource group that will be created for this specific cluster. All VMs and cluster resources reside in this resource group.
```

Comment:
Language doesn't feel super clear, how about something like:

```suggestion
* The Azure Red Hat OpenShift management resource, which represents the ARO cluster, will be placed in the resource group denoted by the  `-g` parameter.  Multiple Azure Red Hat OpenShift management resource can co-exist in the same resource group if you wish.
* All the resources associated with the cluster itself will be placed in the resource group denoted by the `--cluster-resource-group` parameter.  This resource group must not exist at creation time and will be created by the service.  Only one ARO cluster can exist in a given cluster resource group.
```

Reply:
Sounds good. I updated the first bullet slightly b/c per MSFT branding we're going to spell out ARO.